### ISSUE TITLE
Using Laravel Dusk with better-phpunit

### ISSUE THREAD

**Issue by kpyorala** [calebporzio/better-phpunit#82]
## Summary
Using better-phpunit extension with Laravel Dusk tests

## Detailed Description
I'm using this extension to Laravel development and it uses Laravel/Dusk for browser testing. It's a wrapper around PHPUnit but it needs to be started with command `php artisan dusk` instead of `phpunit`

## Possible Implementation Ideas
I have locally edited version of this extension and hard coded way to make this work was just adding this to `phpunit-command.js` files `get binary()` method before return statement:

```js
if (vscode.window.activeTextEditor.document.fileName.indexOf('tests/Browser') > -1) {
  return 'php artisan dusk';
}
```

Final solution should obviously have a setting to turn the feature off/on and better way to detect whether test is using Dusk or not, but you get the idea.

--

If you have any interest of adding this feature to your extension, I can develop my idea idea further and add a PR.

**Comment by calebporzio** [comment]
YESSSS, this is beautiful. I would LOVE this in there. Thanks for being willing to PR this!

**Comment by kpyorala** [comment]
Ok, I'll try to find time next week to do first version for you to check out :)

**Comment by fkraefft** [comment]
@kpyorala any update on this? Would be nice to have this feature :) 

Also, I didn't get where did you hardcore the code. ELI5 please?

**Comment by azurinspire** [comment]
@fkraefft thanks to Caleb's Livewire I hardly use any JS anymore and haven't use Dusk since long time. It seems that code has changed quite a bit since I did my modification but I did it locally to this function:

https://github.com/calebporzio/better-phpunit/blob/81447bd3c0478d6577d697b398578068f53c31d7/src/phpunit-command.js#L69
